scripts/resolve_suggestions_urls.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import json
+from os import path, getcwd
+from jsonpath_rw import parse
+from jsonpointer import set_pointer
+
+UNFORMATTED_SUGGESTIONS_URL_ROOT = (
+    "https://cdn.eq.census-gcp.onsdigital.uk/data/{version}/{region}/{language}"
+)
+VERSION = "v4.0.0"
+
+
+def json_path_to_json_pointer(json_path):
+    """
+    Convert a json path string into a json pointer string.
+    :param json_path: the input data to search
+    :return: json pointer equivalent to the json path
+    """
+    json_pointer = json_path.replace("[", "").replace("]", "").replace(".", "/")
+    return f"/{json_pointer}"
+
+
+def resolve_schema(schema_filepath, suggestions_url_root=None):
+    with open(schema_filepath, "r+") as schema_file:
+        schema_json = json.load(schema_file)
+        json_path = parse("$..suggestions_url")
+        for match in json_path.find(schema_json):
+            json_pointer = json_path_to_json_pointer(str(match.full_path))
+            suggestions_url = match.value
+            if suggestions_url_root:
+                set_pointer(
+                    schema_json,
+                    json_pointer,
+                    suggestions_url.format(suggestions_url_root=suggestions_url_root),
+                )
+            else:
+                set_pointer(schema_json, json_pointer, "")
```

Comment:
This doesn't omit the key, only the value.

Reply:
Not entirely sure what you're getting at here, but this has been updated nonetheless

Reply:
Ah, think I understand the requirement now, the key needs to be completely removed.